HealthcareSystemFrontends/GraphicalEditor/Service/MapObjectServices.cs

```diff
@@ -71,5 +73,56 @@ public List<MapObject> GetNeighboringRoomsForRoom(MapObject room)
             }
             return neighboringRooms;
         }
+
+        private bool CheckIfMapObjectsOverlap(MapObject firstMapObject, MapObject secondMapObject)
+        {
+           for(double coordinateX = Canvas.GetLeft(firstMapObject.Rectangle); coordinateX <= (Canvas.GetLeft(firstMapObject.Rectangle) + firstMapObject.Rectangle.Width); coordinateX += firstMapObject.Rectangle.Width/5)
+            {
+                if (CheckIfPointInsideMapObject(new Point(coordinateX, Canvas.GetTop(firstMapObject.Rectangle)) , secondMapObject))
+                {
+                    return true;
+                }
+            }
```

Comment:
So this wouldn't really fix that much, but maybe get rid of the brackets? And doing <code>if (condition) return true</code>, just do <code>return condition</code>.

Reply:
Thank you for your suggestions! I will apply them in the next PR tomorrow.